StartEycaActivationOrchestrator/index.ts

```diff
@@ -7,12 +7,17 @@ import * as t from "io-ts";
 import { readableReport } from "italia-ts-commons/lib/reporters";
 import { FiscalCode } from "italia-ts-commons/lib/strings";
 
-import { ActivityInput } from "../SuccessEycaActivationActivity/handler";
+import { ActivityInput as StoreEycaExpirationActivityInput } from "../StoreEycaExpirationActivity/handler";
+import { ActivityInput as SuccessEycaActivationActivityInput } from "../SuccessEycaActivationActivity/handler";
+
+import { ShortDate } from "../generated/eyca-api/ShortDate";
 import { ActivityResult } from "../utils/activity";
 import { trackException } from "../utils/appinsights";
 import { internalRetryOptions } from "../utils/retry_policies";
 
 export const OrchestratorInput = t.interface({
+  activationDate: ShortDate,
+  expirationDate: ShortDate,
```

Comment:
It should be `Timestamp` or even `Date`

Reply:
Done in [#30fb002](https://github.com/pagopa/io-functions-cgn/pull/27/commits/30fb002e32ae2a5436af016e2f7d52633bd4017c)